lib/browser/guest-window-manager.ts

```diff
@@ -59,6 +60,23 @@ export function openGuestWindow ({ embedder, guest, referrer, disposition, postD
     }
   }
 
+  if (createWindow) {
+    const webContents = createWindow({
+      webContents: guest,
+      ...browserWindowOptions
+    });
+
+    webContents.loadURL(url, {
+      httpReferrer: referrer,
+      ...(postData && {
+        postData,
+        extraHeaders: formatPostDataHeaders(postData as Electron.UploadRawData[])
+      })
+    });
+
+    return undefined;
```

Comment:
It looks like nothing ever uses the result of `openGuestWindow`, so let's change the return type to `void` and make this just
```suggestion
    return;
```